src/main/java/core/basesyntax/serviceimpl/FileWriterImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+
+    @Override
+    public void writeToFile(String fileName, String data) {
+        try {

```

Comment:
You've used try-with-resources correctly. This ensures that resources are properly closed after the operation is completed, which is good practice.